src/main/java/mate/controller/car/AddCarController.java

```diff
@@ -0,0 +1,40 @@
+package mate.controller.car;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Manufacturer;
+import mate.service.CarService;
+import mate.service.ManufacturerService;
+
+@WebServlet(urlPatterns = "/car/add")
```

Comment:
```suggestion
@WebServlet(urlPatterns = "/cars/add")
```
It is good practice to use plurals. Don't forget to change this link in jsp as well